l10n_it_fiscalcode/readme/HISTORY.rst

```diff
@@ -0,0 +1,14 @@
+[ The change log. The goal of this file is to help readers
+  understand changes between version. The primary audience is
+  end users and integrators. Purely technical changes such as
+  code refactoring must not be mentioned here.
+
+  This file may contain ONE level of section titles, underlined
+  with the ~ (tilde) character. Other section markers are
+  forbidden and will likely break the structure of the README.rst
+  or other documents where this fragment is included. ]
+
+14.0.1.0.1 (2024-05-31)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* [IMP] Add codicefiscale.isvalid() to improve fiscalcode validation
```

Comment:
La versione è sbagliata, attualmente la versione del modulo è 14.0.1.0.2 (vedi  https://github.com/OCA/l10n-italy/blob/14.0/l10n_it_fiscalcode/__manifest__.py#L9).
Teoricamente la versione indicata qui dovrebbe essere 14.0.1.0.3, a meno che non venga fatto il merge di altre PR prima di questa.
 
Tra l'altro come diceva anche @francesco-ooops (https://github.com/OCA/l10n-italy/pull/4180#issuecomment-2145171409) non se se abbia senso aggiungere un HISTORY qui (i commit da indicare poi sarebbero due in caso).

Reply:
Ho rimosso il file come suggerito da @francesco-ooops nel suo commento.

Reply:
> Ho rimosso il file come suggerito da @francesco-ooops nel suo commento.

Però l'hai rimosso dal secondo commit.
Adesso risulta che dopo il primo commit c'è un HISTORY con la v. 16, eliminato poi nel secondo commit.
Dovresti eliminarlo nel primo commit.

Reply:
Primo commit
https://github.com/OCA/l10n-italy/pull/4180/commits/5b5b4b632945349227ec8e85b38b4c1369c59a66

Secondo commit:
https://github.com/OCA/l10n-italy/pull/4180/commits/f9c8d79a01fa9fe19418df65e8e9d09ea0be8ae0

Reply:
Dovrei aver sistemato